sdk/rust-examples/file-seek/Cargo.toml

```diff
@@ -0,0 +1,8 @@
+[package]
+name = "file-seek"
+version = "0.1.0"
+edition = "2021"
```

Comment:
Can you set the `version` to `0.3.0`, add the `author` field and set it to `The Veracruz Development Team`, and add a `description` field with a brief description of what the example is showing?  Moreover, can you set the `edition` for Rust to `2018`, rather than `2021`?